README.md

```diff
@@ -63,6 +63,19 @@ cmake ..
 cmake --build . -j $(sysctl -n hw.ncpu)
 ```
 </details>
+<details><summary>NetBSD</summary>
+
+### Installing dependencies
+```
+pkgin install cmake SDL2_mixer SDL2_ttf libsodium
+```
+### Compiling
+```
+cd build
```

Comment:
If cmake is reasonably recent on NetBSD, `cd` can be avoided:

```bash
cmake -S. -Bbuild ..
cmake --build build -j $(sysctl -n hw.ncpu)
```

Reply:
Cool. It should be the most recent cmake release.